sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeHealthcareAsyncClient.java

```diff
@@ -132,21 +137,22 @@
                 (activationResponse, pollingContext) ->
                     monoError(logger, new RuntimeException("Use the `beginCancelHealthcareJob` to cancel the job")),
                 fetchingOperationIterable(resultId -> Mono.just(new PagedIterable<>(getHealthcareFluxPage(resultId,
-                    finalIncludeStatistics == null ? false : finalIncludeStatistics, context))))
+                    finalTop, finalSkip, finalIncludeStatistics, context))))
             );
         } catch (RuntimeException ex) {
             return PollerFlux.error(ex);
         }
     }
 
-    PagedFlux<HealthcareTaskResult> getHealthcareFluxPage(UUID jobID, boolean showStats, Context context) {
+    PagedFlux<HealthcareTaskResult> getHealthcareFluxPage(UUID jobID, Integer top, Integer skip, Boolean showStats,
+        Context context) {
         return new PagedFlux<>(
-            () -> getPage(null, jobID, showStats, context),
-            continuationToken -> getPage(continuationToken, jobID, showStats, context));
+            () -> getPage(null, jobID, top, skip, showStats, context),
+            continuationToken -> getPage(continuationToken, jobID, top, skip, showStats, context));
     }
 
-    Mono<PagedResponse<HealthcareTaskResult>> getPage(String continuationToken, UUID jobID,
-        boolean showStats, Context context) {
+    Mono<PagedResponse<HealthcareTaskResult>> getPage(String continuationToken, UUID jobID, Integer top, Integer skip,
+        Boolean showStats, Context context) {
```

Comment:
It should be Object Boolean instead of primitive boolean, leave the default value to serivce.  service endpoint takes Boolean.

Reply:
I don't think from an API design it would make sense to have a potential null positional argument. If this argument could be null/ignored why not have it in the options then?

I think I have seen Java API's wanting to expose primitives whenever possible.
@JonathanGiles Do you know if we have a strong opinion on  this?

Reply:
The explored parameter is primitive boolean. If user has option as null, we are using the null for the default value. 
https://github.com/Azure/azure-sdk-for-java/pull/17267/files#diff-5ec2c21f81551de87633be791778c4b0897048d1148892267fa4bf442c44db38R123